src/ti-tle.ts

```diff
@@ -0,0 +1,26 @@
+import { html, css, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+@customElement('ti-tle')
```

Comment:
would prefer it to be `title_field`

Reply:
`field` kind of infers that it is an input.
i've also been working on more concise naming, it's not perfect but i think it conveys its responsibility.
will hold, thanks.